pkg/install/v1alpha1/cstor_volume.go

```diff
@@ -656,6 +661,7 @@ spec:
     {{- $replicaAntiAffinity:= .TaskResult.creategetpvc.replicaAntiAffinity -}}
     {{- $preferredReplicaAntiAffinity:= .TaskResult.creategetpvc.preferredReplicaAntiAffinity }}
     {{- $isClone := .Volume.isCloneEnable | default "false" -}}
+    {{- $zvolWorkers := .Config.ZvolWorkers.value | default "" -}}
```

Comment:
I feel this will error out since ZvolWorkers is not defined as a property of Config

Reply:
I got your point to define under DefaultConfig in  create-volume CastTemplate but may be its not required as i am setting the default value already here ?

I have tested w/o setting any thing with plane SC and its works .